controllers/articles.controller.js

```diff
@@ -18,4 +18,19 @@ exports.getArticles = (request, response, next) => {
         response.status(200).send({ articles: rows });
     })
     .catch(next);
+};
+
+exports.patchArticleVotes = (request, response, next) => {
+    const articleId = request.params.article_id;
+    const voteCount = request.body.inc_votes;
+    return selectArticle(articleId)
+    .then(({ rows }) => {
+        return (rows.length === 0)
+            ? Promise.reject({ status: 404, msg: 'not found' })
```

Comment:
Checking the length of the rows and deciding whether to reject the promise should be the job of the model